plugins/lookup/onepassword.py

```diff
@@ -30,6 +30,10 @@
         aliases: ['vault_password']
       section:
         description: Item section containing the field to retrieve (case-insensitive). If absent will return first match from any section.
+      domain:
+        description: Domain of 1Password. Default is U(1password.com).
+        version_added: 3.2.0
+        default: '1password.com'
```

Comment:
```suggestion
        default: '1password.com'
        type: str
```
(The type should always be mentioned in docs.)